pkg/http/api/v1/handler_test.go

```diff
@@ -453,6 +503,8 @@ func TestRequestHandler_GetMetadata(t *testing.T) {
   ],
   "properties": {
     "harbor.scanner-adapter/scanner-type": "os-package-vulnerability",
+    "harbor.scanner-adapter/vulnerability-database-next-update-at": "2020-03-18 05:00:44 +0000 UTC",
```

Comment:
We should stick to ISO 8601 as specified by the pluggable scanners API

Reply:
Fixed https://github.com/aquasecurity/harbor-scanner-trivy/pull/87/commits/ac84f1aa7a40d624bd3ecea0730ff1c800125135